app/components/trainees/record_actions/view.html.erb

```diff
@@ -1,2 +1,7 @@
-<%= govuk_button_link_to "Record training outcome", edit_trainee_outcome_details_outcome_date_path(trainee), { class: "govuk-!-margin-bottom-0" }  %>
+<%= govuk_button_link_to "Record training outcome", edit_trainee_outcome_details_outcome_date_path(trainee),
+                         { class: "govuk-!-margin-bottom-0" }  %>
+
+<%= govuk_link_to "Defer", trainee_deferral_path(@trainee), class: "defer"  %>
```

Comment:
The styling issue here that @edwardhorsford  spotted should be fixed if you merge in master now.

Reply:
This is styled correctly now